src/common/index.js

```diff
@@ -0,0 +1,7 @@
+export { default as Avatar } from './Avatar';
```

Comment:
Where is this file used?

I think we should avoid the pattern of moving imports to another file for convenience